brew installs breaking helm package - fails in 3.5.2 -> 3.5.3

### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

https://gist.github.com/zrma/c7e415205303c171ec6e68936d336a2d

### 

- [ ] The `brew doctor` above contains no "Warning" lines.

### What were you trying to do (and why)?

brew update

for updating helm

I got some warning during brew doctor but I think it has no relation with this error.
thank you.

### What happened (include all command output)?

```
$ brew upgrade
==> Upgrading 1 outdated package:
helm 3.5.2 -> 3.5.3
==> Upgrading helm 3.5.2 -> 3.5.3
==> Cloning https://github.com/helm/helm.git
Updating /home/zrma/.cache/Homebrew/helm--git
==> Checking out tag v3.5.3
HEAD is now at 041ce5a Fix the example for --time-format flag
HEAD is now at 041ce5a Fix the example for --time-format flag
==> make build
Last 15 lines from /home/zrma/.cache/Homebrew/Logs/helm/01.make:
2021-03-11 23:55:14 +0900

make
build

GO111MODULE=on go build  -tags '' -ldflags '-w -s -X helm.sh/helm/v3/internal/version.version=v3.5.3 -X helm.sh/helm/v3/internal/version.metadata= -X helm.sh/helm/v3/internal/version.gitCommit=041ce5a2c17a58be0fcd5f5e16fb3e7e95fea622 -X helm.sh/helm/v3/internal/version.gitTreeState=dirty -X helm.sh/helm/v3/pkg/lint/rules.k8sVersionMajor=1 -X helm.sh/helm/v3/pkg/lint/rules.k8sVersionMinor=20 -X helm.sh/helm/v3/pkg/chartutil.k8sVersionMajor=1 -X helm.sh/helm/v3/pkg/chartutil.k8sVersionMinor=20' -o '/tmp/helm-20210311-2054-125wm4q/bin'/helm ./cmd/helm
/home/zrma/.cache/Homebrew/go_mod_cache/pkg/mod/github.com/moby/moby@v17.12.0-ce-rc1.0.20200618181300-9dc6525e6118+incompatible/pkg/system/path.go:9:2: missing go.sum entry for module providing package github.com/containerd/continuity/pathdriver (imported by github.com/docker/docker/pkg/system); to add:
        go get github.com/docker/docker/pkg/system@v1.4.2-0.20200203170920-46ec8731fbce
make: *** [Makefile:78: /tmp/helm-20210311-2054-125wm4q/bin/helm] Error 1

READ THIS: https://docs.brew.sh/Troubleshooting
```

### What did you expect to happen?

update helm successfully

### Step-by-step reproduction instructions (by running `brew` commands)

$ brew upgrade
$ brew update

Also seeing this, here's my log. Hope it helps.
```zsh
Updated 1 tap (homebrew/core).
==> New Formulae
qt-libiodbc                  qt-mariadb                   qt-mysql                     qt-percona-server            qt-postgresql                qt-unixodbc                  xray
==> Updated Formulae
helm ✔                 atomicparsley          esptool                git-annex              k3d                    menhir                 python@3.9             terraform              zeek
abcmidi                aws-elasticbeanstalk   exploitdb              git-town               kubevela               nqp                    rakudo                 tomcat@9
angular-cli            awscurl                flintrock              glibmm                 lego                   nuclei                 scipy                  txr
ansible-lint           caf                    flow-cli               helmsman               libtommath             php-cs-fixer           scw                    wireshark
aqbanking              calicoctl              fortio                 hledger                libvoikko              pnpm                   serverless             xrootd
atkmm                  credstash              gel                    infracost              libxkbcommon           prometheus-cpp         step                   zbar

You have 1 outdated formula installed.
You can update it with brew upgrade.
==> Upgrading 1 outdated package:
helm 3.5.2 -> 3.5.3
==> Upgrading helm 3.5.2 -> 3.5.3
==> Cloning https://github.com/helm/helm.git
Cloning into '/home/zageron/.cache/Homebrew/helm--git'...
==> Checking out tag v3.5.3
HEAD is now at 041ce5a Fix the example for --time-format flag
==> make build
Last 15 lines from /home/zageron/.cache/Homebrew/Logs/helm/01.make:
go: downloading github.com/docker/spdystream v0.0.0-20160310174837-449fdfce4d96
go: downloading github.com/PuerkitoBio/purell v1.1.1
go: downloading github.com/prometheus/client_golang v1.7.1
go: downloading github.com/mailru/easyjson v0.7.0
go: downloading github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578
go: downloading github.com/prometheus/common v0.10.0
go: downloading github.com/prometheus/client_model v0.2.0
go: downloading github.com/prometheus/procfs v0.2.0
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading github.com/beorn7/perks v1.0.1
go: downloading github.com/cespare/xxhash/v2 v2.1.1
go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369
/home/zageron/.cache/Homebrew/go_mod_cache/pkg/mod/github.com/moby/moby@v17.12.0-ce-rc1.0.20200618181300-9dc6525e6118+incompatible/pkg/system/path.go:9:2: missing go.sum entry for module providing package github.com/containerd/continuity/pathdriver (imported by github.com/docker/docker/pkg/system); to add:
        go get github.com/docker/docker/pkg/system@v1.4.2-0.20200203170920-46ec8731fbce
make: *** [Makefile:78: /tmp/helm-20210311-4658-g66wmp/bin/helm] Error 1

READ THIS: https://docs.brew.sh/Troubleshooting

These open issues may also help:
brew installs breaking helm package - fails in 3.5.2 -> 3.5.3 https://github.com/Homebrew/linuxbrew-core/issues/22543
```

^ exact same issue - full trace:

```
==> Upgrading helm 3.5.2 -> 3.5.3
==> Downloading https://linuxbrew.bintray.com/bottles/go-1.16.2_1.x86_64_linux.bottle.tar.gz
==> Downloading from https://akamai.bintray.com/ac/ac0418ec93d70e6f4a38a6f6a99ca49f4d323cf01e88bb14e807fb2ddbc225d5?__gda__=exp=1615813112~hmac=070e5c
######################################################################## 100.0%
==> Cloning https://github.com/helm/helm.git
Cloning into '/home/user/.cache/Homebrew/helm--git'...
==> Checking out tag v3.5.3
HEAD is now at 041ce5a Fix the example for --time-format flag
==> Installing dependencies for helm: go
==> Installing helm dependency: go
==> Pouring go-1.16.2_1.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/go/1.16.2_1: 9,944 files, 482.7MB
==> Installing helm
==> make build
Last 15 lines from /home/user/.cache/Homebrew/Logs/helm/01.make:
go: downloading github.com/go-openapi/jsonreference v0.19.3
go: downloading github.com/go-openapi/swag v0.19.5
go: downloading github.com/PuerkitoBio/purell v1.1.1
go: downloading github.com/mailru/easyjson v0.7.0
go: downloading github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578
go: downloading github.com/prometheus/client_model v0.2.0
go: downloading github.com/prometheus/common v0.10.0
go: downloading github.com/beorn7/perks v1.0.1
go: downloading github.com/cespare/xxhash v1.1.0
go: downloading github.com/prometheus/procfs v0.2.0
go: downloading github.com/cespare/xxhash/v2 v2.1.1
go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.2-0.20181231171920-c182affec369
/home/user/.cache/Homebrew/go_mod_cache/pkg/mod/github.com/moby/moby@v17.12.0-ce-rc1.0.20200618181300-9dc6525e6118+incompatible/pkg/system/path.go:9:2: missing go.sum entry for module providing package github.com/containerd/continuity/pathdriver (imported by github.com/docker/docker/pkg/system); to add:
	go get github.com/docker/docker/pkg/system@v1.4.2-0.20200203170920-46ec8731fbce
make: *** [Makefile:78: /tmp/helm-20210315-7679-1qxrv6a/bin/helm] Error 1

READ THIS: https://docs.brew.sh/Troubleshooting

These open issues may also help:
brew installs breaking helm package - fails in 3.5.2 -> 3.5.3 https://github.com/Homebrew/linuxbrew-core/issues/22543
```

Tried `go get github.com/docker/docker/pkg/system@v1.4.2-0.20200203170920-46ec8731fbce` as mentioned on the trace and no dice.

--- a/Formula/helm.rb
+++ b/Formula/helm.rb
@@ -17,6 +17,7 @@ class Helm < Formula
   depends_on "go" => :build
 
   def install
+    system "go", "mod", "tidy"
     system "make", "build"
     bin.install "bin/helm"
 

